### ISSUE TITLE
$pug->requirements() should return true for cache if the cache is set to false.

### ISSUE THREAD

**Issue by esiao** [pug-php/pug#75]
Setting the cache option to false when using the requirements throws:

```
2 requirements are missing.
The cache folder does not exists, please enter in a command line : mkdir -p .
The cache folder is not writable, please enter in a command line : chmod -R +w .
```

I would like to enable the cache only on production and staging environments but can't unless the requirements test is also done on these environments.

**Comment by kylekatarnls** [comment]
Seem fair, I will fix that.

**Comment by kylekatarnls** [comment]
Hi, you can test the fix with an upgrade to 2.4.5.

**Comment by esiao** [comment]
Works, thanks 👍
